### DISCUSSION TITLE
[Discord Server] Create a discord server for the devs/contributors.

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [pear-devs/pear-desktop#1299] [Ideas]
Currently there is no easy way to coordinate on what a contributor should work on, other than GitHub issues and PRs.
Whilst they may serve a purpose, they aren't always ideal when it comes to simple stuff like:

- *You forgot to add a dev dependency for "builtin-modules"*
- *You forgot to add a dependency for "@cliqz/adblocker-electron-preload"*

(yes, you did forget to add those)

etc

I don't want to make an issue for stuff like that.
Plus we can have better discussions on discord.

**Comment by JellyBrick** [comment]
I (currently) don't have any plans to create a Discord Server.
We are already using GitHub discussions, reviews, and Actions (CI), so I believe we can address the issues mentioned earlier.

**Reply by ArjixWasTaken** [reply]
That is understandable, its just that I can't talk as casually on GitHub discussions.
So I am less active unless a project also has a discord server.

**Reply by JellyBrick** [reply]
hmm, okay, I'll consider this suggestion.

**Reply by JellyBrick** [reply]
@th-ch https://discord.gg/bSzQKkGV
